core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -43,6 +46,47 @@ THE SOFTWARE.
         <l:task href="${rootURL}/${it.viewUrl}newJob" icon="icon-new-package icon-md" it="${app}" permission="${it.itemCreatePermission}" title="${%NewJob(it.newPronoun)}"/>
       </st:include>
 
+      <l:task href="#modal" icon="icon-new-package icon-md" title="Remodal Popup"/>
+      <!-- <st:include page="popup.jelly" optional="true" /> -->
+      
+      <div class="remodal new-job-popup" data-remodal-id="modal" id="add-item-panel">
```

Comment:
I probably don't know how to do it yet. Do you have ideas how?
I always used library styles by referring to their class and though this is how they should be used.